packages/contracts-js/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "Apache-2.0",
   "repository": "https://github.com/thirdweb-dev/js/tree/main/packages/contracts-js",
   "dependencies": {
-    "@thirdweb-dev/contracts": "3.1.7"
+    "@thirdweb-dev/contracts": "3.1.8-0"
```

Comment:
What does this change? Being able to override the trusted forwarder for pack?

Also can you run yarn build from the root package to get the new ABIs checked in?

Reply:
yes the initialize function is now accepting _trustedForwarders array, so the signature changed